services/src/workflows/workflow.rs

```diff
@@ -22,6 +22,7 @@ impl WorkflowId {
 #[derive(Clone, Debug, Serialize, Deserialize, ToSchema)]
 pub struct Workflow {
     #[serde(flatten)]
+    #[schema(value_type = crate::api::model::operators::TypedOperator)]
```

Comment:
isnt it dangerous to assume the two types have the same schema? shouldn't Workflow be in api/mode?

Reply:
Maybe but then we also are not able to use custom schema impls instead of derives.
We will notice it if the integration tests from the libs won't work.
Some types are not optimal yet, but it is cumbersome to make everything very good.